pkg/services/ngalert/store/instance_database.go

```diff
@@ -64,6 +64,7 @@ func (st DBstore) ListAlertInstances(ctx context.Context, cmd *models.ListAlertI
 			params = append(params, p...)
 		}
 
+		// TODO: Why do we use the title here?
```

Comment:
I see no reason for having the title in the response.

Reply:
That saves us a join in the query, too :-)